dss/error.py

```diff
@@ -37,6 +36,26 @@ def __init__(self, title: str = "User is not authorized to access this resource"
                          title,
                          *args, **kwargs)
 
+
+def include_retry_after_header(return_code, method, uri):
+    uuid_pattern = '[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}'
+    bundle_checkout_pattern = f'/v1/bundles/{uuid_pattern}/checkout'
+    bundle_checkout = re.compile(bundle_checkout_pattern)
```

Comment:
If we're going to use regex, it should probably be compiled once per Lambda cold start.

The uuid pattern can also be pulled in from `dss/storage/identifiers.py`

Reply:
I just discovered that the uuid pattern in `dss/storage/identifiers.py` doesn't work, but the one I copy-pasted from the swagger.yml does.  >.<